src/vstart.sh

```diff
@@ -36,13 +36,14 @@ if [ -n "$VSTART_DEST" ]; then
     SRC_PATH=`(cd $SRC_PATH; pwd)`
 
     CEPH_DIR=$SRC_PATH
-    CEPH_BIN=${PWD}/bin
-    CEPH_LIB=${PWD}/lib
+    CEPH_BIN=${CEPH_BIN:-${PWD}/bin}
+    CEPH_LIB=${CEPH_LIB:-${PWD}/lib}
 
     CEPH_CONF_PATH=$VSTART_DEST
     CEPH_DEV_DIR=$VSTART_DEST/dev
     CEPH_OUT_DIR=$VSTART_DEST/out
     CEPH_ASOK_DIR=$VSTART_DEST/asok
+    CEPH_OUT_CLIENT_DIR=$CEPH_OUT_DIR/client
```

Comment:
As this change will also impact non-dashboard users, would about keeping the default setting (`$CEPH_OUT_DIR`), but, allowing users to tweak it (`$CEPH_OUT_CLIENT_DIR`)?
```suggestion
    CEPH_OUT_CLIENT_DIR=${CEPH_OUT_CLIENT_DIR:-$CEPH_OUT_DIR}
```

Reply:
then that would mean setting this directory on our jobs explicitly right?

Reply:
yeah! the idea was not to change the legacy default behaviour of vstart, but allow to customize it when required.

Reply:
fair enough. I'll update it.

Reply:
done! can you also check the `export CEPH_OUT_CLIENT_DIR` that I am doing is correct. and do I need to modify it in any other tests?